src/ansi-c/expr2c.cpp

```diff
@@ -1998,14 +1918,12 @@ std::string expr2ct::convert_constant_bool(bool boolean_value)
 {
   // C doesn't really have these
   if(boolean_value)
-    return "TRUE";
+    return "1";
```

Comment:
Changing the boolean value was the original reason for this commit.

I am sorry for the reformatting around this change in the commit.

My intend was to change TRUE to 1 and FALSE to 0.